module/shared.py

```diff
@@ -98,9 +98,14 @@ def check_log(update: Update, command_name: str, is_query: bool = False):
         DbManager.insert_into(table_name="stat_list", values=(command_name, chat_id, date.today()))
 
     if config_map['debug']['disable_chatid_logs'] == 0:
-        with open("logs/chatid.txt", "r+") as r_log:
-            log = r_log.read()
-        if not str(chat_id) in log:
+        try:
+            with open("logs/chatid.txt", "r+") as r_log:
+                log = r_log.read()
+        except FileNotFoundError:
+            open("logs/chatid.txt", "w+")
+            log = ()
```

Comment:
....?

Reply:
The following lines check if the current chat_id is in log. 
Usually, log would be the whole record read from the file, but in the case the file is just been created it gets initialized to an empty tuple, which will not contain the chat_id